src/stan-users-guide/gaussian-processes.Rmd

```diff
@@ -1086,3 +1151,98 @@ generated quantities {
   Omega = L_Omega * L_Omega';
 }
 ```
+
+## Supported Covariance Functions
+
+Currently, Stan supports several build in covariance functions (kernels). These are the squared exponential, also known as the radial basis function or the gaussian kernel, a dot product or linear covariance function, the periodic covariance function, and three additional special cases of the Matern covariance function, that are the Matern 5/2 kernel, Matern 3/2 and the Matern 1/2 covariance function, or the exponential covariance function. We will review functional form, implementation and provide a visualization with simulated data below.
+
+### The Squared Exponential
+
+A common choice for a covariance function is the Squared Exponential covariance function. This is also known as the Radial Basis Function or the Gaussian kernel, due to the fact that its function resembles a gaussian. The functional form is as follows
+
+$$
+K(x, x' \mid \sigma, l) = \sigma^2 \exp\left(-\dfrac{||x - x'||_2^2}{2 l^2}\right).
+$$
+for $\sigma,l \geq 0$, where $||\cdot||_2^2$ is the squared l-2 norm, and $x$ and $x'$ are observations of design matrix $X$. The $K_{i,j}$ will then be the covariance computed between rows $i$ and $j$ of the design matrix. Here's what draws from the squared exponential kernel look like, before and after being conditioned on data, with fixed hyper priors for the magntitude parameter $\sigma^2$ and length-scale parameter $l$:
+
+//////////////GRAPHIC
+
+We can sample from a multivariate gaussian distribution in Stan by first instantiating a covariance matrix, for example for `vector[D] x[N]`, and parameters `real<lower=0> sigma` and `real<lower=0> length_scale` as
+
+```
+matrix[N, N] K = gp_exp_quad_cov(x, sigma, l)
+```
+
+And then we add noise and sample from a multivariate gaussian distribution using cholesky decomposition:
+
+```
+
+transformed parameters {
+  vector[N] f;
+  matrix[N, N] L_K;
+  K = add_diag(K, epsilon);
+  L_K = choleskey_decompose(K);
+  f = L_K * eta;
+}
+...
+model {
+  eta ~ normal(0, 1);
+}
+```
+Where since $\eta \sim normal(0, 1)$, $f$ is distributed multivariate normal.
```

Comment:
"$f$ is distributed multivariate normal."
-> $f$ has a multivariate normal prior with covariance matrix $K$.